From 77932d7e96eba3437eb3d8e5321ed688b7c8689a Mon Sep 17 00:00:00 2001
From: Olivier Blin <olivier.blin@softathome.com>
Date: Fri, 28 Apr 2023 00:48:40 -0700
Subject: [PATCH] Make libgbm and libdrm conditional dependencies
 https://bugs.webkit.org/show_bug.cgi?id=255393
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Reviewed by Žan Doberšek.

Like in 248522@main for GTK.

* Source/cmake/OptionsWPE.cmake:
- Make libgbm and libdrm conditional dependencies

Canonical link: https://commits.webkit.org/263491@main
---
 Source/cmake/OptionsWPE.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Source/cmake/OptionsWPE.cmake b/Source/cmake/OptionsWPE.cmake
index b21959469f88..72cde472c229 100644
--- a/Source/cmake/OptionsWPE.cmake
+++ b/Source/cmake/OptionsWPE.cmake
@@ -359,7 +359,6 @@ SET_AND_EXPOSE_TO_BUILD(HAVE_ACCESSIBILITY ${ENABLE_ACCESSIBILITY})
 SET_AND_EXPOSE_TO_BUILD(USE_ATSPI ${ENABLE_ACCESSIBILITY})
 SET_AND_EXPOSE_TO_BUILD(USE_CAIRO TRUE)
 SET_AND_EXPOSE_TO_BUILD(USE_EGL TRUE)
-SET_AND_EXPOSE_TO_BUILD(USE_GBM TRUE)
 SET_AND_EXPOSE_TO_BUILD(USE_GCRYPT TRUE)
 SET_AND_EXPOSE_TO_BUILD(USE_LIBEPOXY TRUE)
 SET_AND_EXPOSE_TO_BUILD(USE_LIBWPE TRUE)
-- 
2.34.1

